src/scripts/manage_artifacts/clean_for_integration_test.py

```diff
@@ -56,9 +54,6 @@ def main() -> None:
     if not args.bucket_prefix or args.bucket_prefix[-1] != "/":
         raise ValueError("Bucket prefix must be provided and end with a '/'")
 
-    if "main" in args.bucket_prefix:
```

Comment:
Ah I forgot about us having to remove this if we want to clean up `main`. Hmm, should we add another safeguard?
```
if `dev` not in `args.bucket`:
   raise ValueError("Must be using buckets in the develop environment")
```

OR

```
dev_buckets = [`recover-dev-input-data`, 'recover-dev-intermediate-data`, 'recover-dev-processed-data', 'recover-dev-cloudformation']

if `args.bucket` not in dev_buckets:
   raise ValueError("Must be using buckets in the develop environment")
```
Overkill?

Reply:
We will also be running this on the prod buckets too - But the `staging` directory of those buckets. I think controlling this in the workflow makes the most sense here.

Reply:
We could add a safeguard for prod/main (we still want to be able to clean up prod/staging). I think this script should be agnostic to the object key, except for the aforementioned case.

Reply:
How does this guard condition look?

https://github.com/Sage-Bionetworks/recover/pull/126/commits/6e366260fdaa97dffded87acc1ecadde13787c1b

Reply:
Sorry, my brain is not functioning right now. Yes I second what Phil said.